crates/model/Cargo.toml

```diff
@@ -9,22 +9,21 @@ license = "MIT OR Apache-2.0"
 doctest = false
 
 [dependencies]
-anyhow = "1"
-bigdecimal = "0.3"
-chrono = { version = "0.4", default-features = false, features = ["serde", "clock"] }
-derivative = "2.2"
-ethabi = "15.0"
-enum-utils = "0.1"
```

Comment:
replaced with `strum`. We already use it elsewhere and it seamed like the more well-established crate.